hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug.robot

```diff
@@ -15,22 +15,52 @@
 
 *** Settings ***
 Documentation       Test ozone Debug CLI
-Library             OperatingSystem
-Resource            ../commonlib.robot
-Test Timeout        2 minute
-Suite Setup         Write key
-*** Variables ***
+Library             Collections
+Resource            ../lib/os.robot
 
 *** Keywords ***
-Write key
-    Execute             ozone sh volume create o3://om/vol1 --space-quota 100TB --namespace-quota 100
-    Execute             ozone sh bucket create o3://om/vol1/bucket1
-    Execute             ozone sh key put o3://om/vol1/bucket1/debugKey /opt/hadoop/NOTICE.txt
+Execute read-replicas CLI tool
+    Execute                         ozone debug read-replicas o3://om/${VOLUME}/${BUCKET}/${TESTFILE}
+    ${directory} =                  Execute     find /opt/hadoop -maxdepth 1 -name '${VOLUME}_${BUCKET}_${TESTFILE}_*' | tail -n 1
```

Comment:
I think the output here should be sorted  based on timestamp so that the right directory is obtained.

Reply:
Even though the directories are sorted by the creation time if they are ordered by name (the directories end with a timestamp), I looked after and find is not always consistent with the order. I changed it to use ls as it will be consistent with the order: "Sort entries alpha‐betically if none of -cftuvSUX nor --sort is specified.".
But I think the CI didn't fail because of this. The stopped datanode didn't become STALE and DEAD as the interval were 5 and 10 minutes. I changed that and added a test case for the STALE and one for the DEAD datanode case.

Reply:
Thanks for clarifying @dombizita .